share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -767,8 +767,74 @@ ml-linux-x86_64-rocm-build:
       job: ml-linux-x86_64-rocm-generate
 
 ########################################
-# Machine Learning - Darwin aarch64 (MPS)
+# Machine Learning - Linux aarch64 (CPU)
 ########################################
+.ml-linux-aarch64-cpu:
+  extends: [ ".linux_aarch64" ]
```

Comment:
Should these also specify the amazonlinux image?

Reply:
What's the correct syntax for that? I just copied the template from other ML pipelines, I have absolutely no idea what any of it means.

P.S. Feel free to push to my branch if you want, or suggest edits on this PR.